### PR TITLE
send reminder message immediately when appointment is created

### PR DESCRIPTION
**Pull request by clark-teeple**
Line 42 requires PR #19 .

It's probably time to think about breaking the "send message by appointment" stuff into its own module, since it now exists in some form here, in the sendReminders handler and in the POST message handler.

The template currently will be the same for all 3 (appt creation, day before, day of) reminder texts.

### CODE REVIEW

File: src/api/handlers/appointments/postAppointments.js [hunk 1]
```diff
@@ -24,9 +28,52 @@ export default (req, res) => {
...
+    timeSent: moment().format("YYYY-MM-DD HH:mm:ss")
+  };
+
+  sendMessage(patientPhoneNumber, messageBody)
+    .then(async () => {
+      let messageId;
+      try {
+        messageId = await db("messages").insert(message);
+      } catch (error) {
+        res.status(500).send({
```
Comment (iamstevedavis) [inline src/api/handlers/appointments/postAppointments.js:61]: If you aren't returning here does it continue execution and try and do the res.status(200)?

Comment (clark-teeple) [reply src/api/handlers/appointments/postAppointments.js:61]: whoops. yup.

Comment (clark-teeple) [reply src/api/handlers/appointments/postAppointments.js:61]: updated.

Comment (clark-teeple) [reply src/api/handlers/appointments/postAppointments.js:61]: Is this better:
```
  await sendMessage(patientPhoneNumber, messageBody)
    .catch((error) => {
      res.status(500).send({ error, message: "Failed to send appointment reminder"});
      return;
    });

  let messageId;
  try {
    messageId = await db("messages").insert(message);
  } catch (error) {
    res.status(500).send({
      error,
      message: "Could not save message to database"
    });
    return;
  }
  res.status(200).send({
    insertedAppointmentId,
    messageId: messageId[0]
  });
  return;
```

**Pull request merged**